### PR TITLE
List disabled

### LINKED ISSUE #353: 🚀 Feature: Add a cli command to get list of disabled playbooks
### 🔖 Feature description

Add CLI option `list-disabled` with alias `ld`.

Usage:
`node ./app/main.js --list-disabled` or `node ./app/main.js -ld`

This option to read all playbooks and list out all disabled playbooks. 

### 🎤 Pitch

Currently, disabled playbooks require manual tracking. This command will make it easy to just list all disabled playbooks in one go and then someone can check if they can be updated and enabled again.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/drph4nt0m/avbot-charts/blob/HEAD/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by nguyenhung15913**
# Description
- Added in new option <code>-ld, --list-disabled</code>
- Added codes to list out all disabled playbooks (<code>enable: false</code>) inside <code>main()</code>

Fixes #353 

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- Test: <code>node ./app/main.js -ld</code> will return <code>https://www.amcharts.com/visited_countries/#KR,MM</code>

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation/readme
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

If the function does not return your expected out, please feel free to ask me for any correction.

### CODE REVIEW

File: app/main.js [hunk 1]
```diff
@@ -18,17 +18,36 @@ commander.program
   .option('-a, --all', 'Scrape all airports in all the countries')
   .option('-p, --prod', 'Run in production mode')
   .option('-cm, --completed-map', 'World map link of implemented playbooks')
+  .option('-ld,--list-disabled', 'List out all disabled playbooks')
   .parse(process.argv);
 
 const options = commander.program.opts();
 
+//Function to list out all disabled playbooks (enable: false)
+const listDisabled = (playbooksDir) => {};
```
Comment (drph4nt0m) [inline app/main.js:27]:
```suggestion
```

File: app/main.js [hunk 2]
```diff
@@ -18,17 +18,36 @@ commander.program
   .option('-a, --all', 'Scrape all airports in all the countries')
   .option('-p, --prod', 'Run in production mode')
   .option('-cm, --completed-map', 'World map link of implemented playbooks')
+  .option('-ld,--list-disabled', 'List out all disabled playbooks')
   .parse(process.argv);
 
 const options = commander.program.opts();
 
+//Function to list out all disabled playbooks (enable: false)
```
Comment (drph4nt0m) [inline app/main.js:26]:
```suggestion
```

File: app/main.js [hunk 3]
```diff
@@ -18,17 +18,36 @@ commander.program
...
 
   const playbooksDir = `${process.cwd()}/playbooks`;
 
   let prodMode = false;
 
+  //List disabled playbooks
+  if (options.listDisabled) {
+    try {
+      const files = fs.readdirSync(playbooksDir);
+      files.forEach((file) => {
```
Comment (drph4nt0m) [inline app/main.js:40]:
```suggestion
      logger.info(`Disabled playbooks:`, { type: 'general' });
      files.forEach((file) => {
        const fsPlaybook = JSON.parse(fs.readFileSync(`${playbooksDir}/${file}`, 'utf8'));
        if (fsPlaybook.enabled === false) {
          logger.info(`[${fsPlaybook.country.iso}] ${fsPlaybook.country.name}`, { type: 'general' });
        }
      });
```

File: app/main.js [hunk 4]
```diff
@@ -18,17 +18,36 @@ commander.program
...
+        const fsPlaybook = JSON.parse(fs.readFileSync(`${playbooksDir}/${file}`, 'utf8'));
+        if (fsPlaybook.enabled === false) {
+          console.log(file);
+        }
+      });
```
Comment (drph4nt0m) [inline app/main.js:45]:
```suggestion
      logger.info(`Disabled playbooks:`, { type: 'general' });
      files.forEach((file) => {
        const fsPlaybook = JSON.parse(fs.readFileSync(`${playbooksDir}/${file}`, 'utf8'));
        if (fsPlaybook.enabled === false) {
          logger.info(`[${fsPlaybook.country.iso}] ${fsPlaybook.country.name}`, { type: 'general' });
        }
      });
```

File: app/main.js [hunk 5]
```diff
@@ -18,17 +18,36 @@ commander.program
...
   .option('-p, --prod', 'Run in production mode')
   .option('-cm, --completed-map', 'World map link of implemented playbooks')
+  .option('-ld,--list-disabled', 'List out all disabled playbooks')
   .parse(process.argv);
 
 const options = commander.program.opts();
 
+//Function to list out all disabled playbooks (enable: false)
+const listDisabled = (playbooksDir) => {};
+
```
Comment (drph4nt0m) [inline app/main.js:28]:
```suggestion
```

**Review by drph4nt0m — CHANGES_REQUESTED** [review]
@nguyenhung15913 can you make these changes please. Thanks!

**Comment by nguyenhung15913** [comment]
I fixed. Can you check @drph4nt0m ?

**Review by drph4nt0m — APPROVED** [review]

**Comment by drph4nt0m** [comment]
@nguyenhung15913 everything looks good. Can you just fix the linting error with the comments and then I will merge the changes.

File: app/main.js (hunk 5 above)
Comment (drph4nt0m) [reply app/main.js:28]: Please remove this unused function

File: app/main.js [hunk 6]
```diff
@@ -18,17 +18,37 @@ commander.program
...
 async function main() {
   logger.debug(process.argv.join(' '), { type: 'general' });
 
   const playbooksDir = `${process.cwd()}/playbooks`;
 
   let prodMode = false;
 
+  //List disabled playbooks
```
Comment (drph4nt0m) [inline app/main.js:36]: Fix the linting on the comment. You can use `npm run lint`and find what is going wrong.

**Review by drph4nt0m — CHANGES_REQUESTED** [review]

File: app/main.js (hunk 6 above)
Comment (nguyenhung15913) [reply app/main.js]: Hi. Last time I fixed that issue with new commit and somehow it did not go through to this pull request. I made another one. Can you please check @drph4nt0m ? Thanks

**Comment by drph4nt0m** [comment]
@nguyenhung15913 thanks for your contribution. Looking forward to more PRs.

**Pull request merged**